tg/tdlib.py

```diff
@@ -113,6 +113,10 @@ def reply_message(
 
         return self._send_data(data)
 
+    def search_contacts(self, target: str) -> AsyncResult:
+        data = {"@type": "searchChats", "query": target, "limit": 10}
```

Comment:
Let's pass a limit as optional argument.
In the future we might make it as an official tdlib client.

Reply:
done